src/main/java/teammates/ui/newcontroller/FeedbackSessionInfo.java

```diff
@@ -380,10 +432,32 @@ public void validate() {
     }
 
     /**
-     * The basic request body format for saving of feedback session.
+     * The request body format for saving of feedback session.
      */
     public static class FeedbackSessionSaveRequest extends FeedbackSessionBasicRequest {
```

Comment:
If by now this has not been implemented, what is the usage of this?

Reply:
It is used in `SaveFeedbackSessionAction`. I just fix the typo I introduced in the previous PR :P